tools/logger/logger.c

```diff
@@ -141,6 +143,7 @@ static int configure_uart(const char *file, unsigned int baud)
 
 int main(int argc, char *argv[])
 {
+	static const char opt_frm[] = "ho:i:l:ps:c:u:tev:rdLf:F:g:";
 	struct convert_config config;
```

Comment:
Please introduce opt_frm in a separate patch? And why is it static?

Reply:
I can't see any profit after removing `static` keyword

Reply:
The profit will be the code clarity. Static variables are rarely used inside functions and when used they have a special purpose.

Reply:
`static const` is in general better approach, because then variable is initialized only once - in opposite to `const` which will be initialized in each function call. For `main` function there is actually no difference, but we should keep good practice. Moreover after removing `static` checkpatch complain: `WARNING: const array should probably be static const`